src/front-end/src/helper-methods.js

```diff
@@ -68,10 +89,17 @@ export function resetPassword(emailAddress) {
   auth
     .sendPasswordResetEmail(emailAddress)
     .then(() => {
+      window.alert('We sent you an email with a link to reset your password.');
       console.log('Email sent');
     })
     .catch((error) => {
       console.log(error);
+      switch (error.code) {
+        case 'auth/invalid-email':
+          break;
+        default:
+          break;
+      }
```

Comment:
This code doesn't do anything. If you meant to print the error, you can update it. Otherwise, you can delete the whole switch statement.

Reply:
Before I added this in, if a user hit the 'Confirm' button even without adding any input, the 'We sent you an email with a link to reset your password' popup would be displayed. I added the switch statement to catch that, so that the red input error will be displayed (so the break does as I intended). I'll also add in the case for when an email address is entered which isn't in our DB (auth/user-not-found)

Reply:
My mistake then, I was wrong!